styles.css

```diff
@@ -1,826 +1,860 @@
-  color: #253858;
-}
-
-li.fram {
-  font-family: Poppins, sans-serif;
-  height: 38px;
-  width: 36px;
-  font-style: normal;
-  font-weight: 100;
-  font-size: 20px;
-  line-height: 24px;
-  margin-top: 1rem;
-  margin-bottom: 0;
-  margin-left: 2rem;
-  color: #000;
-}
-
-li.skills {
-  font-family: Poppins, sans-serif;
-  height: 210px;
-  width: 36px;
-  font-style: normal;
-  font-weight: 100;
-  font-size: 20px;
-  line-height: 24px;
-  margin-top: 1rem;
-  margin-left: 2rem;
-  color: #000;
+  gap: 16px;
+  width: 303px;
+  height: 66.2px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  margin: 12px;
 }
 
-/* ---------Form Section------------ */
+/* -------- Form Section ---------- */
 
-form {
+.container-form {
   display: flex;
   flex-direction: column;
-}
-
-.contact-form {
-  margin-top: 10rem;
+  align-items: center;
+  gap: 20px;
+  width: 375px;
   background-color: #6070ff;
-  background-repeat: no-repeat;
-  background-position-x: 350px;
-  padding: 80px 10px 10px;
-  border-top-left-radius: 120px;
-  text-align: center;
+  padding: 114px 24px;
+  border-top-left-radius: 75px;
 }
 
-.contact-content h1 {
-  font-family: Poppins, sans-serif;
-  color: #fff;
-  text-align: center;
-  height: 56px;
-  font-size: 40px;
-  line-height: 56px;
-  margin: 20px 0;
+.contact-content {
+  width: 295px;
 }
 
-.contact-content p {
-  font-family: Poppins, sans-serif;
-  font-style: normal;
-  color: #ebebff;
-  height: 136px;
+p.contact {
+  font-weight: 400;
   font-size: 20px;
   line-height: 28px;
-  font-weight: 4000;
+  color: #ebebff;
   text-align: center;
 }
 
-.name-container input {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
+div.name-container,
+div.mail-container {
+  width: 327px;
   height: 48px;
-  padding-left: 5px;
 }
 
-.mail-container input {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
+div.name-container input,
+div.mail-container input {
+  font-weight: 400;
   font-size: 17px;
-  line-height: 20px;
   color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
-  height: 48px;
-  padding-left: 5px;
+  line-height: 20px;
+  width: 273px;
+  height: 18px;
+  padding: 15px 38px 15px 16px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
 }
 
-.textarea-container textarea {
-  font-family: Poppins, sans-serif;
-  border: 1px solid transparent;
-  border-radius: 8px;
-  background: #fff;
-  font-size: 17px;
-  line-height: 20px;
-  color: #172b4d;
-  width: 80%;
-  margin-top: 20px;
+.textarea-container {
+  width: 327px;
   height: 170px;
-  padding-left: 5px;
 }
 
-button {
+textarea {
+  width: 273px;
+  height: 140px;
+  padding: 15px 38px 15px 16px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+}
+
+button.btn-get {
```

Comment:
Great job team 🤝

- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), kindly implement all button interactions for the  `Get In Touch button.

Reply:
Hello @Nejjari001 

- [ ] The last point in the project requirements suggests that you implement button interactions of `hover` and `enable`.